### PR TITLE
 Extend styleci config to check more code style rules

### PR DESCRIPTION
**Pull request by acrobat**
Use the recommended preset to add some more code style checks.

Btw pr/issue 666 😱

### CODE REVIEW

File: .styleci.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-preset: psr2
+preset: recommended
 
-enabled:
-  - return
+disabled:
+  - align_double_arrow
+  - concat_without_spaces
```
Comment (GrahamCampbell) [inline .styleci.yml]: Do you want to also turn on `concat_with_spaces`?

File: .styleci.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+  - no_multiline_whitespace_before_semicolons
```
Comment (GrahamCampbell) [inline .styleci.yml]: There is a less aggressive version of this fixer you might want to replace it with. I can't remember it's name, but I think it does something along the lines of removing the whitespace, but not in multiline contexts.

**Comment by GrahamCampbell** [comment]
Also, might be an idea to hold off on these changes, until the PHP 7 PR has gone through, or not to commit the CS changes in this PR?

**Comment by acrobat** [comment]
I thought it would be good to do these CS changes here and let the php7 pr rebase when this is merged. This way that pr has less changes, more relevant changes and makes it easier to review imo.

**Comment by GrahamCampbell** [comment]
Since the changes in this PR are autogenerated, I think this one should wait tbh.

File: .styleci.yml (hunk 1 above)
Comment (acrobat) [reply .styleci.yml]: yes let's do that!

File: .styleci.yml (hunk 2 above)
Comment (acrobat) [reply .styleci.yml]: Do you mean this one "no_singleline_whitespace_before_semicolons" ?

Comment (GrahamCampbell) [reply .styleci.yml]: Yeh, that sounds right. :)

Comment (acrobat) [reply .styleci.yml]: So I keep this one disabled and enable the singleline one, right?

**Comment by acrobat** [comment]
@Nyholm any remarks/objections before merging this? I thinks we agreed on splitting the php 7 pr, so this one can be merged before.

**Pull request merged**